styleguide/derek/_partials/solutions/leadSplit.ejs

```diff
@@ -0,0 +1,13 @@
+<!-- Lead pattern -->
+<!--
+.lead class naming not used in alignment with typical pattern structure
+instead here .lead-wrapper is used to avoid having to override bootstrap base class
+ -->
+<div class="lead-wrapper">
+  <div class="lead-headerWrapper">
+    <h1 class="lead-header left">Title goes here</h1>
```

Comment:
Actually, the issue is that it needs to be inside a subpanel well.  This is straight from production: 

![image](https://cloud.githubusercontent.com/assets/5263371/23669619/6b361362-032a-11e7-940e-bbd5e0737b47.png)